src/guide/essentials/event-handling.md

```diff
@@ -357,3 +357,7 @@ The `.exact` modifier allows control of the exact combination of system modifier
 - `.middle`
 
 These modifiers restrict the handler to events triggered by a specific mouse button.
+
+::: tip Note
```

Comment:
**suggestion**: we're trying to reduce the number of highlighted blocks and use them only if absolutely necessary. Would you mind removing the wrapping `tip` and keeping the added information?

Reply:
Sure, but moved the emphasis to the beginning of the sentence. Or would it be better to just keep "`.left`" as the beginning?

Reply:
@midskyey the changed emphasis looks good to me 👍🏻